relay-server/src/utils/unreal.rs

```diff
@@ -212,9 +212,35 @@ fn merge_unreal_context(event: &mut Event, context: Unreal4Context) {
         os_context.name = Annotated::new(os_major);
     }
 
-    if let Some(gpu_brand) = runtime_props.misc_primary_gpu_brand.take() {
+    // See https://github.com/EpicGames/UnrealEngine/blob/5.3.2-release/Engine/Source/Runtime/RHI/Private/DynamicRHI.cpp#L368-L376
+    if let Some(adapter_name) = context.engine_data.get("RHI.AdapterName") {
```

Comment:
Does it make sense to have a fallback to `runtime_props.misc_primary_gpu_brand` if the is not available?

E.g. with a quick google search I found a report [here](https://forums.unrealengine.com/t/cannot-open-epic-games-launcher-on-m1-macbook-air/482663/10) which does not contain `EngineData`

Reply:
Agreed, added a fallback to `runtime_props.misc_primary_gpu_brand`